src/libs/CurrencyUtils.ts

```diff
@@ -23,56 +25,45 @@ Onyx.connect({
  * For currencies that have decimal places > 2, floor to 2 instead:
  * https://github.com/Expensify/App/issues/15878#issuecomment-1496291464
  *
- * @param {String} currency - IOU currency
- * @returns {Number}
+ * @param currency - IOU currency
```

Comment:
I dont know if this is still needed

Reply:
Yes, param is needed, but we don't type them, just a name and description.